src/reuse/_comment.py

```diff
@@ -447,6 +448,17 @@ class VimCommentStyle(CommentStyle):
     INDENT_AFTER_SINGLE = " "
 
 
+class XQueryCommentStyle(CommentStyle):
+    """XQuery comment style."""
+
+    _shorthand = "xq"
```

Comment:
I would call this `xquery`

Reply:
thank you for your feedback, @carmenbianca; additional commit(s) incoming!